.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "python.pythonPath": "backend\\env\\Scripts\\python.exe",
+    "python.pythonPath": "env\\Scripts\\python.exe",
```

Comment:
This entire folder should be ignored.

Reply:
sure, final PR could exclude these files. They were in master so I inherited them.